lib/moon_web/components/left_menu.ex

```diff
@@ -23,7 +23,7 @@ defmodule MoonWeb.Components.LeftMenu do
       :on-click={@click}
       id="left-menu-container"
       class={
-        "hidden fixed h-screen lg:flex lg:flex-shrink-0 w-80 flex-col z-[10000]",
```

Comment:
sure mobile site menu is not broken with this change?

Reply:
For me it seems to work the same as moon.io, also the mobile view.